.github/scripts/helm-test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Keptn Lifecycle Toolkit Helm Testing
+#
+# This script supports the comparison of standard values and expected templated results to helm chart
+# it is used to make sure changes to the chart are intentional and produce expected outcomes
+
+echo "running simple tests"
+  tests=$(find ./.github/scripts/.tests -maxdepth 1 -mindepth 1 -type d )
+
+  errors=0
+  successful=0
+
+  for test in $tests
+  do
```

Comment:
let's also add a log output stating which test case is being executed

Reply:
great point!